_ruby_libs/pages.rb

```diff
@@ -206,6 +207,9 @@ def initialize(site, parent_page, path, data)
 
   def add_child_page(page)
     self.data['child_pages'] << page
+    if page.data['indexed']
+      self.data['indexed_child_pages'] << page
```

Comment:
Are all child pages of indexed pages indexed or do you need to check that the child page is also indexed before adding it?

Reply:
That's what `if page.data['indexed']` checks for (not the same as `self.data['indexed']`). Though an indexed child page of an un-indexed (parent) page won't show.

Not sure if we should `assert` or try propagate the child up to the closest ancestor that is indexed. We are doing neither.

Reply:
Sorry. No I misread the page.data check as self.data. Looks good.